optuna/pruners/hyperband.py

```diff
@@ -49,22 +49,40 @@ class HyperbandPruner(BasePruner):
         Thus, for example, if ``HyperbandPruner`` has :math:`4` pruners in it,
         at least :math:`4 \\times 10` pruners are consumed for startup.
 
+    Example:
+
+        We minimize an objective function with Hyperband pruning algorithm.
+
+        .. testcode::
+
+            from optuna import create_study
+            from optuna.pruners import HyperbandPruner
+
+            def objective(trial):
+                x = trial.suggest_uniform('x', -1, 1)
+                return x ** 2
```

Comment:
The pruner is never activated because this objective function does not have `trial.report` or `trial.should_prune`. How about using the same objective function as the other pruners (e.g., https://github.com/optuna/optuna/blob/v1.2.0/optuna/pruners/median.py#L14-L44).